### PR TITLE
feat: show inactive window on quit warning

### LINKED ISSUE #307: Show inactive windows on quit warning
I often close apps with `cmd+tab` pressing `q` over an app I want to close. At the moment, the quit warning shows, but the app isn't brought to the front.

Here's an example of how I'd like it to work
show-on-quit

### PR DESCRIPTION
**Pull request by erickzhao**
Fixes #307.

Implements #308, but uses a new IPC event instead of `remote`.

### CODE REVIEW

File: src/main/windows.ts [hunk 1]
```diff
@@ -61,6 +63,11 @@ export function createMainWindow(): Electron.BrowserWindow {
     shell.openExternal(url);
   });
 
+  ipcMainManager.on(IpcEvents.SHOW_INACTIVE, () => {
+    console.info('erick was here');
```
Comment (felixrieseberg) [inline src/main/windows.ts]: 😄

Comment (felixrieseberg) [inline src/main/windows.ts]:
```suggestion
    console.info('erick was here');
```

File: tests/main/windows-spec.ts [hunk 2]
```diff
@@ -95,5 +97,12 @@ describe('windows', () => {
       (getOrCreateMainWindow().webContents as any).emit('will-navigate', e);
       expect(e.preventDefault).toHaveBeenCalled();
     });
+
+    it('shows the window on IPC event', () => {
+      const w = getOrCreateMainWindow();
+      ipcMainManager.emit(IpcEvents.SHOW_INACTIVE);
+      expect(w.showInactive).toHaveBeenCalled();
+
```
Comment (felixrieseberg) [inline tests/main/windows-spec.ts]:
```suggestion
```

File: src/main/windows.ts (hunk 1 above)
Comment (erickzhao) [reply src/main/windows.ts]: 🙈

**Pull request merged**